core/src/avm2/globals.rs

```diff
@@ -906,12 +906,6 @@ pub fn load_player_globals<'gc>(
 
     // package `flash.geom`
     class(activation, flash::geom::matrix::create_class(mc), script)?;
-    avm2_system_class!(
```

Comment:
Since `SystemClasses::rectangle` exists, we should add `rectangle` to `avm2_system_classes_playerglobal` invocation, right?

Reply:
Right. I went ahead and removed `SystemClasses::rectangle`, `SystemPrototypes::rectangle`, `SystemClasses::point`, and `SystemPrototypes::point`, which are now unused.